index.d.ts

```diff
@@ -61,6 +57,8 @@ declare class FirstChunkStream extends DuplexStream {
 		options: FirstChunkStream.Options,
 		transform: FirstChunkStream.TransformFunction
 	);
+
+	static readonly stop: FirstChunkStream.StopSymbol;
```

Comment:
This needs to be documented in the readme too. And it needs a doc comment here about what it does, and a usage example.